Bottom: 3f550255741e236cf2f686fb4dc687ba5b2c59dd
Top:    8c0f07589e8e09b4a48466fe208ef164177adcb6
Author: Ian Kent <ikent@redhat.com>
Date:   2013-10-02 09:24:32 +0800

Refresh of autofs-5.0.7-setup-program-map-env-from-macro-table.patch

---

diff --git a/include/macros.h b/include/macros.h
index 5077b5d..5a5486e 100644
--- a/include/macros.h
+++ b/include/macros.h
@@ -40,6 +40,6 @@ void macro_free_global_table(void);
 void macro_free_table(struct substvar *table);
 const struct substvar *
 macro_findvar(const struct substvar *table, const char *str, int len);
-void macro_setenv(const struct substvar *table);
+void macro_setenv(struct substvar *table);
 
 #endif
diff --git a/lib/macros.c b/lib/macros.c
index 33c2ada..d7c392f 100644
--- a/lib/macros.c
+++ b/lib/macros.c
@@ -422,7 +422,7 @@ macro_findvar(const struct substvar *table, const char *str, int len)
 }
 
 /* Set environment from macro variable table */
-void macro_setenv(const struct substvar *table)
+void macro_setenv(struct substvar *table)
 {
 	const struct substvar *sv = system_table;
 	const struct substvar *lv = table;
